Goobi/src/de/sub/goobi/metadaten/Paginator.java

```diff
@@ -99,9 +99,20 @@ private void assertSelectionIsNotNull() {
 		}
 	}
 
+	/**
+	 * Throws NumberFormatException if `paginationStartValue` isn’t a valid
+	 * number of the type specified by `paginationType`.
+	 * 
+	 * @throws NumberFormatException
+	 *             if `paginationStartValue` isn’t valid
+	 */
 	private void assertValidPaginationStartValue() {
 		// arabic numbers
 		if (paginationType == Paginator.Type.ARABIC) {
+			/* coverity[USELESS_CALL]
+			 * Integer.parseInt() throws NumberFormatException if
+			 * paginationStartValue cannot be parsed to an `int`, and this is
+			 * what we want here. */
```

Comment:
Would adding `throws NumberFormatException` to the function declaration also fix the coverity warning (instead of muting coverity by this comment)?